### PR TITLE
Introducing a skeleton structure for the iOS library

### PR DESCRIPTION
**Pull request by davidair**

### CODE REVIEW

**Review by AndreasRicci — APPROVED** [review]

File: ios/FirebaseContinue/Example/FirebaseContinue/main.m [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//
```
Comment (paulb777) [inline ios/FirebaseContinue/Example/FirebaseContinue/main.m:1]: Copyright

File: ios/FirebaseContinue/Example/Podfile.lock [hunk 2]
```diff
@@ -0,0 +1,45 @@
+PODS:
```
Comment (paulb777) [inline ios/FirebaseContinue/Example/Podfile.lock]: Podfile.lock should be in .gitignore and not checked. (They may be checked in for apps, but should not be for libraries)

File: ios/FirebaseContinue/Example/Tests/Tests.m [hunk 3]
```diff
@@ -0,0 +1,35 @@
+//
```
Comment (paulb777) [inline ios/FirebaseContinue/Example/Tests/Tests.m:1]: copyright

File: ios/FirebaseContinue/Example/Tests/Tests-Prefix.pch [hunk 4]
```diff
@@ -0,0 +1,7 @@
+//  The contents of this file are implicitly included at the beginning of every test case source file.
```
Comment (paulb777) [inline ios/FirebaseContinue/Example/Tests/Tests-Prefix.pch]: Delete or add copyright

File: ios/FirebaseContinue/FirebaseContinue.podspec [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  s.summary          = 'Firebase Continue enables mobile developers to integrate activity transitioning from their mobile apps to the web.'
+
+  s.description      = <<-DESC
+Firebase Continue for iOS enables iOS developers to easily integrate activity transitioning
+from their apps to the web by way of Chrome extensions.
+                       DESC
+
+  s.homepage         = 'https://github.com/firebase/firebase-continue'
+  s.license          = { :type => 'Apache 2.0', :file => '../../LICENSE' }
+  s.author           = { 'Andreas Ricci' => 'ricciandreas@gmail.com' }
```
Comment (paulb777) [inline ios/FirebaseContinue/FirebaseContinue.podspec]: Should be  s.authors          = 'Google, Inc.'

File: ios/FirebaseContinue/Example/FirebaseContinue/FirebaseContinue-Prefix.pch [hunk 6]
```diff
@@ -0,0 +1,16 @@
+//
```
Comment (paulb777) [inline ios/FirebaseContinue/Example/FirebaseContinue/FirebaseContinue-Prefix.pch:1]: Delete or add copyright

File: ios/FirebaseContinue/Example/FirebaseContinue.xcworkspace/contents.xcworkspacedata [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (paulb777) [inline ios/FirebaseContinue/Example/FirebaseContinue.xcworkspace/contents.xcworkspacedata]: *.xcworkspace should be in .gitignore

**Review by paulb777 — CHANGES_REQUESTED** [review]

File: ios/FirebaseContinue/Example/FirebaseContinue/main.m (hunk 1 above)
Comment (davidair) [reply ios/FirebaseContinue/Example/FirebaseContinue/main.m:1]: done

File: ios/FirebaseContinue/Example/Tests/Tests.m (hunk 3 above)
Comment (davidair) [reply ios/FirebaseContinue/Example/Tests/Tests.m:1]: done

File: ios/FirebaseContinue/Example/FirebaseContinue/FirebaseContinue-Prefix.pch (hunk 6 above)
Comment (davidair) [reply ios/FirebaseContinue/Example/FirebaseContinue/FirebaseContinue-Prefix.pch:1]: done

File: ios/FirebaseContinue/Example/Podfile.lock (hunk 2 above)
Comment (davidair) [reply ios/FirebaseContinue/Example/Podfile.lock]: done

File: ios/FirebaseContinue/Example/Tests/Tests-Prefix.pch (hunk 4 above)
Comment (davidair) [reply ios/FirebaseContinue/Example/Tests/Tests-Prefix.pch]: done

File: ios/FirebaseContinue/FirebaseContinue.podspec (hunk 5 above)
Comment (davidair) [reply ios/FirebaseContinue/FirebaseContinue.podspec]: done

**Review by paulb777 — CHANGES_REQUESTED** [review]
And git rm the files now in the .gitignore

**Review by paulb777 — APPROVED** [review]

**Pull request merged**